packages/utils/src/lib/reports/generate-md-report.ts

```diff
@@ -41,25 +41,29 @@ export function generateMdReport(
   report: ScoredReport,
   commitData: CommitData | null,
 ): string {
+  const { categories = [] } = report;
+  const reportWithCategories = { ...report, categories };
+  const printCategories = categories.length > 0 ? true : false;
```

Comment:
Can be simplified for same reasons as in `generate-stdout-summary.ts`:

```suggestion
  const printCategories = categories.length > 0;
```

Then you can just use the original `report` below instead of the extra `reportWithCategories` variable.

Reply:
Simplified.